ntbs-service/Migrations/20191010150705_UpdateTravelAndVisitorEntitiesWithAdditionalFieldsAndFixedNavigation.cs

```diff
@@ -0,0 +1,365 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace ntbs_service.Migrations
+{
+    public partial class UpdateTravelAndVisitorEntitiesWithAdditionalFieldsAndFixedNavigation : Migration
```

Comment:
I tried for this one, fought with EF for a long time before settling with this... I'll give it another shot after tackling the other points in the review.

Reply:
cool, i'd be happy to pair on this if helpful :)

Reply:
Second attempt also failed - discussed moving to use rebase instead of merge from master in the future - which would remove issues with sandwiched migrations.